sw/device/silicon_creator/lib/crypto/ecdsa_p256/meson.build

```diff
@@ -2,7 +2,8 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-sw_lib_crypto_otbn_sources += {
+# Create otbn sources dictionary
+sw_lib_crypto_otbn_sources = {
```

Comment:
Since this is only going to include local files, you can name it `sw_lib_crypto_ecdsa_p256_src` or similar. The prefix in the variable is used because Meson does not have support for local variables in this context.